cfep-16.md

```diff
@@ -0,0 +1,49 @@
+
+<table>
+<tr><td> Title </td><td> Setting license and license_family field </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Isuru Fernando &lt;isuruf@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> March 28, 2020</td></tr>
+<tr><td> Updated </td><td> March 28, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> https://github.com/conda-forge/conda-smithy/pull/1268 and https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/36 </td></tr>
+</table>
+
+## Abstract
+
+This document proposes that we enforce the use SPDX license expressions
+in license field in package metadata and to hotfix the package metadata
```

Comment:
How do you propose to do the hotfix? Is this a bot thing, or a repodata patch? Or what is the thought process here?

Reply:
A hotfix as in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/36

Reply:
That seems like a rather large hammer for this, and easy to get wrong in a lot of cases.  Wouldn't it be better / more consistent to just enforce the SPDX indentifiers going forward?

The inconsisency will come in that since we cannot change the artifacts, the metadata in the artifact won't have the same license_family entry as repodata.json.

Reply:
IIUC the patch is only applied to packages that have an SPDX compatible `license` entry. Otherwise it doesn't do anything.

Reply:
Ahh, yeah. That would be OK, if `add` is really add and not `overwrite`, which is how I read this sentence

Reply:
`license` is enforced to be SPDX. `license_family` can have only a predefined set of values and is not SPDX.
What I'm proposing is that `license_family` be added automatically when missing by parsing the `license` which is in SPDX.

I pushed a clarification. Feel free to suggest/push any changes to make it more clear.